tests/functional/configuration/test_all_options.py

```diff
@@ -612,6 +615,7 @@ class TestWorkerConfig(ConfigurationTestSuite):
         "worker-max-trials": 1,
         "worker-max-broken": 8,
         "reservation-timeout": 18,
+        "idle-timeout": 19,
```

Comment:
idle-timeout should also be tested here: https://github.com/Epistimio/orion/pull/684/files#diff-fcc8709fe851b3f4c1151b08c4b72c00f1300440b30d4da082c6740a2874fb73R697

Reply:
my browser does not bring me to the the place your are pointing to

Reply:
Oops, my neither. The changes in code broke the link. Here's in current develop branch: https://github.com/Epistimio/orion/blob/develop/tests/functional/configuration/test_all_options.py#L693

Reply:
done

Reply:
done